litmus-portal/frontend/src/utils/promUtils.ts

```diff
@@ -1,6 +1,11 @@
 /* eslint-disable no-unused-expressions */
-import { ParsedPrometheusData } from '../models/dashboardsData';
-import { PromQuery } from '../models/graphql/dashboardsDetails';
+/* eslint-disable no-useless-escape */
+/* eslint-disable no-param-reassign */
```

Comment:
Can you remove the eslint disable commands if possible?

Reply:
Will try to but, can't remove this `no-param-reassign` as lint wants to return something on iterating but its not my use-case, its a kind of warning but we have purposely used the map/forEach this way. @Saranya-jena